samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
Still not a fan. What is being ignored? The event? The stuff we do inside the event? The default logic after the event? The whole request?

Ignore makes sense for most of your internal uses, but it doesn't make sense in the events. I still think Handle was more accurate for these. I've handled the event, the error, the response, etc..